doc/source/appendices/tutorial.rst

```diff
@@ -112,26 +112,15 @@ Look at the output.  Go to the ICE_RUNDIR (defined in **icepack.settings**). A s
   * How does your output compare to the sample output provided for this release?
 
 
-Add a New Tracer and Run Some Tests
---------------------------------------
-
-In this exercise, add a new tracer associated with fluffballs.
-Call the tracer fluff and make it depend on ice area.  Follow the step-by-step instructions in the :ref:`addtrcr` documentation.
-Once you have implemented the model changes, be sure to add fluffballs output to the standard output diagnostics and turn on the
-fluff tracer.  Then update the timeseries plotting script to plot the fluffballs values over time.
-
-* First, set the initial value, physics, sources, and sinks of fluff to zero and make sure fluff values remain zero throughout the run
-
-* Add some constant atmospheric forcing and review results
-
-* Change the dependency to ice volume, how do the results change?
-
-* Modify the physics to create some physics processes, see isotopes or aerosols for some ideas
+Modify the Configuration or Code
+------------------------------------
 
 * Set Up a Longer Run.  Modify npt in icepack_in.  npt defines the number of timesteps to run.  Details about namelist options are in the documentation (:ref:`case_settings`).
```

Comment:
Remove the capitalization (or add it in the next bullet for consistency).
Format npt as a code variable.

Reply:
Done.